src/App.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import { MainScreen } from './pages';
+import Checkoutpage from './pages/CheckoutPage';
```

Comment:
```suggestion
import { MainScreen, Checkoutpage } from './pages';
```
essa é a forma correta de import com o index.js!

Reply:
Modificação feita